fdbserver/storageserver.actor.cpp

```diff
@@ -4416,6 +4473,7 @@ AddingShard::AddingShard(StorageServer* server, KeyRangeRef const& keys)
 }
 
 void AddingShard::addMutation(Version version, bool fromFetch, MutationRef const& mutation) {
+	server->counters.logicalBytesMoveInOverhead += mutation.expectedSize();
```

Comment:
Is logicalBytesMoveInOverhead defined/declared? 
(Maybe I missed it)

Reply:
Yes, it is defined and declared, let me push my changes, and see if it will be refreshed.

Reply:
yes, it is. Maybe I didn't push my changes, let me push it and see if it can be refreshed.